Assets/Scripts/Controllers/CharacterController.cs

```diff
@@ -6,7 +6,7 @@
 public class CharacterController : MonoBehaviour
 {
     private Rigidbody2D _rb;
-    private Vector2 _movement;
+    [SerializeField] private Vector2 _movement;
```

Comment:
Set equal to a default value:
`private Vector2 _movement = Vector2.zero`

Also, nitpick, but probably rename for clarity:
`private Vector2 _movementInput = Vector2.zero`

Also, it seems like this field shouldn't be serialized? Doesn't seem like people should be setting the movement input value in the inspector. Let me know your thoughts.